packages/contract/src/contract-call-script/src/main.sw

```diff
@@ -1,34 +1,36 @@
 script;
 
-use std::context::registers::*;
+use std::contract_id::ContractId;
 
-const CONTRACT_ID_LEN = 32;
-const WORD_SIZE = 8;
+fn get_script_data<T>() -> T {
+    // This line fixes a bug
```

Comment:
Can you clarify in this comment what bug? Is there an issue tracking it?

Reply:
I don't even know what the bug is :D 

First started realizing the bug here: https://fuellabs.slack.com/archives/C01KBNGSWGJ/p1652788684067789
And reduced it to "some kind of a padding issue" here: https://fuellabs.slack.com/archives/C01KBNGSWGJ/p1652799760675829